galaxy_api/api/ui/urls.py

```diff
@@ -15,6 +16,17 @@
     viewsets.CollectionVersionViewSet,
     basename='collection-versions',
 )
+router.register(
+    'collections/(?P<collection>[0-9a-z_]+/[0-9a-z_]+)/versions/[0-9A-Za-z.+-]+/certified',
+    # 'collections/(?P<collection>{})/versions/(?P<version>{})/certified'.format(
+    #     viewsets.CollectionViewSet.lookup_value_regex,
+    #     viewsets.CollectionVersionViewSet.lookup_value_regex
+    # ),
+    # https://github.com/encode/django-rest-framework/issues/1337
+    # urlpatterns = format_suffix_patterns(urlpatterns)
+    viewsets.CollectionVersionViewSet,
+    basename='collection-versions',
+)
```

Comment:
This should look something more like the following:

```
router.register(
    'collections/(?P<collection>{})/versions/(?P<version>{})/certified'.format(
        viewsets.CollectionViewSet.lookup_value_regex,
        utils.VERSION_REGEXP
    ),
    viewsets.CollectionVersionViewSet,
    basename='collection-versions',
)
```

Reply:
`VERSION_REGEXP` is a standalone object and is not designed to be injected into routing expression.

Reply:
Reusing `CollectionViewSet.lookup_value_regexp` is probably better though.

Reply:
Per conversation in Slack with @cutwater, we don't need to register this at all. It's handled automatically by the `drf_action` decorator.